schema/defs.yaml

```diff
@@ -244,6 +244,10 @@ description: Common schema definitions shared by all themes
         country:
           description: ISO 3166 alpha2 country code
           type: string
+    sourceTags:
+      description: Attributes that pass directly through from the source data
+      type: object
```

Comment:
If it's truly a tags array, it shouldn't be *just* an `object`: should have validation constraints to ensure the keys and values must be strings.

You need something along the lines of https://stackoverflow.com/a/36237572/1911388.